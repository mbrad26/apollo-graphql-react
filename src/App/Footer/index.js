import React from 'react';

import Link from '../../Link';

import './style.css';

const Footer = () => (
  <div className="Footer">
  </div>
);

export default Footer;
