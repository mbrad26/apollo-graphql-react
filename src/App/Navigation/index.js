import React from 'react';
import { Link } from 'react-router-dom';

import './style.css';
import * as routes from '../../constants/routes';

const Navigation = () => (
    <header className='Navigation'>
      <div className='Navigation-link'>
        <Link to={routes.PROFILE}>Profile</Link>
      </div>
      <div className='Navigation-link'>
        <Link to={routes.ORGANIZATION}>Organization</Link>
      </div>
    </header>
)

export default Navigation;
