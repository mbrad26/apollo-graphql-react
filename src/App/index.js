import React from 'react';

import './style.css';
import Profile from '../Profile';

const App = () => {
  return (
    <Profile />
  );
};

export default App;
