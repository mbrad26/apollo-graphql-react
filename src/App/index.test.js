import React from 'react';
import { MockedProvider } from "@apollo/client/testing";
import { render, screen } from '@testing-library/react';

import App from './';

describe('App', () => {
  it('renders snapshot', () => {
    const { container } = render(
      <MockedProvider client={[]}>
        <App />
      </MockedProvider>
    );

    expect(container.firstChild).toMatchSnapshot();
  })
});
