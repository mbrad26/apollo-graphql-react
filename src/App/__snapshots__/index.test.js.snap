// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders snapshot 1`] = `
<div
  class="App"
>
  <header
    class="Navigation"
  >
    <div
      class="Navigation-link"
    >
      <a
        href="/profile"
      >
        Profile
      </a>
    </div>
    <div
      class="Navigation-link"
    >
      <a
        href="/"
      >
        Organization
      </a>
    </div>
  </header>
  <div
    class="App-main"
  >
    <div
      class="App-content_large-header"
    >
      <div>
        Organization
      </div>
    </div>
  </div>
</div>
`;
